src/database/migrations/migrations.js

```diff
@@ -84,41 +83,38 @@ exports.up = (knex, Promise) => {
     table.string('email');
   }),
 
-    knex.schema.createTableIfNotExists(tables.users_tracks, (table) => {
-      logger.debug('Creating users_tracks table.');
-      table.increments('user_track_id').primary();
-      table.integer('user_id');
-      table.integer('track_id');
-    }),
+  knex.schema.createTableIfNotExists(tables.users_tracks, table => {
+    logger.debug('Creating users_tracks table.');
+    table.increments('user_track_id').primary();
+    table.integer('user_id');
+    table.integer('track_id');
+  }),
 
-    knex.schema.createTableIfNotExists(tables.users_artists, (table) => {
-      logger.debug('Creating users_artists table.');
-      table.increments('user_artist_id').primary();
-      table.integer('user_id');
-      table.integer('artist_id');
-    }),
+  knex.schema.createTableIfNotExists(tables.users_artists, table => {
+    logger.debug('Creating users_artists table.');
+    table.increments('user_artist_id').primary();
+    table.integer('user_id');
+    table.integer('artist_id');
+  }),
 
-    knex.schema.createTableIfNotExists(tables.users_users, table => {
-      logger.debug('Creating users_users table.');
-      table.increments('user_friend_id').primary();
-      table.integer('user_id');
-      table.integer('friend_id');
-    }),
-  ]);
-};
+  knex.schema.createTableIfNotExists(tables.users_users, table => {
+    logger.debug('Creating users_users table.');
+    table.increments('user_friend_id').primary();
+    table.integer('user_id');
+    table.integer('friend_id');
+  }),
+]);
 
-exports.down = (knex, Promise) => {
-	return Promise.all([
-    knex.schema.dropTable(tables.tracks),
-    knex.schema.dropTable(tables.albums),
-    knex.schema.dropTable(tables.tracks_rating),
-    knex.schema.dropTable(tables.artists),
-    knex.schema.dropTable(tables.artists_tracks),
-    knex.schema.dropTable(tables.users),
-    knex.schema.dropTable(tables.admins),
-    knex.schema.dropTable(tables.users_tracks),
-    knex.schema.dropTable(tables.users_artists),
-    knex.schema.dropTable(tables.albums_artists),
-    knex.schema.dropTable(tables.users_users),
-  ]);
-};
\ No newline at end of file
+exports.down = (knex, Promise) => Promise.all([
+  knex.schema.dropTable(tables.tracks),
```

Comment:
Lo probe pero fallo con `Knex:warning - migrations failed with error: tables.map is not a function
` :/